setup.py

```diff
@@ -5,6 +5,10 @@
 with open("README.rst", "r", encoding="utf-8") as fh:
     long_description = fh.read()
 
+so3g_etxras = ["so3g"]
+dev_extras = ["pytest", "pytest_twisted", "pytest-docker-compose", "pytest-cov", "coverage"]
```

Comment:
These errors should now be fixed. 

I think I missed the documentation surrounding the use of the requirements files because I had already installed the package using `pip` on the local working copy (since the tests do not work without this), and that uses the requirements encoded in `setup.py`, so I didn't expect there to be a second source of dependency information.

Reply:
Right, perhaps we should ditch the use of `requirements.txt`...